webpack.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  output: {
+    libraryTarget: 'commonjs'
+  },
+  mode: 'production',
+  entry: './js/browser.js',
+  module: {
+    rules: [
+      {
+        test: /\.jsonld$/,
+        loader: 'json-loader'
+      }
+    ]
+  }
```

Comment:
Don't need `.jsonld` rule in this repo.  However, if those schemas were to change to `.yaml`, would use a similar construct for `yaml-loader`.
```suggestion
  entry: './js/browser.js'
```